### ISSUE TITLE
Telegram Desktop repo package and standalone binary broken on Ubuntu 19.10

### ISSUE THREAD

**Issue by jdrch** [telegramdesktop/tdesktop#7481]
### Steps to reproduce
1. Install Telegram Desktop on Ubuntu using `# apt install telegram-desktop` OR download and extract the archive directly from the Telegram website
2. Attempt to launch Telegram by clicking the application shortcut for the repo installation or double-clicking the extracted `Telegram` or `Updater` files from the archive

### Expected behaviour

Telegram Desktop should start.

### Actual behaviour

The window with the error messages below shows instead:

![Screenshot from 2020-03-28 15-12-44](https://user-images.githubusercontent.com/1056759/77832895-4c7a1380-7107-11ea-9899-8dfd233550ef.png)

### Configuration
**Operating system:**

Ubuntu 19.10

**Version of Telegram Desktop:**

1.9.21

**Used theme**:

In the app: dark theme

In Ubuntu: Yaru-dark

**Logs**:

```
[2020.03.28 15:09:58] Executable path before check: /home/judah/Desktop/tsetup.1.9.21/Telegram/Telegram
[2020.03.28 15:09:58] Launched version: 1009021, install beta: [FALSE], alpha: 0, debug mode: [FALSE], test dc: [FALSE]
[2020.03.28 15:09:58] Executable dir: /home/judah/Desktop/tsetup.1.9.21/Telegram/, name: Telegram
[2020.03.28 15:09:58] Initial working dir: /home/judah/Desktop/tsetup.1.9.21/Telegram/
[2020.03.28 15:09:58] Working dir: /home/judah/.local/share/TelegramDesktop/
[2020.03.28 15:09:58] Command line: /home/judah/Desktop/tsetup.1.9.21/Telegram/Telegram -noupdate
[2020.03.28 15:09:58] FATAL: Could not open '/home/judah/.local/share/TelegramDesktop/log_startXX.txt' for writing log!
[2020.03.28 15:09:58] Launcher filename: telegramdesktop.desktop
[2020.03.28 15:09:58] Checking for XDG Desktop Portal...
[2020.03.28 15:09:58] XDG Desktop Portal is present!
[2020.03.28 15:09:58] Usage of XDG Desktop Portal is disabled.
[2020.03.28 15:09:58] Connecting local socket to /run/user/1000/ead271995c29bc1d22f6b72dbda5c892-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2020.03.28 15:09:58] This is the only instance of Telegram, starting server and app...
```

**Comment by Aokromes** [comment]
you must report that to ubuntu maintainer, not here.

**Comment by jdrch** [comment]
@Aokromes 

> ubuntu maintainer

Except the binary downloaded directly from the Telegram site - which is completely independent of Linux distribution - isn't working either. It actually produces the same error message. This would indicate issues on the Telegram side of things, IMO.

**Comment by Aokromes** [comment]
be sure you remove telegram from apt and download beta from https://github.com/telegramdesktop/tdesktop/releases

**Comment by jdrch** [comment]
> be sure you remove telegram from apt 

Yep, did `# apt purge telegram-desktop` already

> and download beta from https://github.com/telegramdesktop/tdesktop/releases

OK, I'll try that, thanks!

**Comment by jdrch** [comment]
> be sure you remove telegram from apt and download beta from https://github.com/telegramdesktop/tdesktop/releases

So that didn't work, either. What *did* solve the problem was deleting the `/home/MyUsername/.local/share/TelegramDesktop/` directory, which had somehow been switched to root permissions. I removed in Nautilus running as root, which I started via `# nautilus`.

Once I deleted the folder `# install telegram-desktop` got everything working again.

Not sure why or how the original problem happened at all.
